### PR TITLE
[NEW] close emoji box using Keyboard Escape key

### LINKED ISSUE #13929: Adding emoji to message: focus in the filter box and make escape work
### Description:

After clicking on the 'add emoji' icon for a message, there are two problems:

* if you want to filter, you first have to click the filter box. The way it is now, the popup is half-way pop-up, half-way pop-under.
* You can't press 'escape' to get rid if the pop-up should you change your mind.

### Expected behavior:

* Since a form control that you can type in pops up, I expect to be able to type in it without extra clicks. 
* Make it close when you press escape.

### Additional context

Firefox, both Linux and Windows.

### PR DESCRIPTION
**Pull request by mohamedar97**
Closes #13929 

added the option to close the emoji box with escape key and added auto focus to emoji search field

### CODE REVIEW

**Comment by mohamedar97** [comment]
@knrt10 is this the change you wanted?

**Comment by knrt10** [comment]
This only solves for package-lock.json. See there is one conflicting girl. You need to fetch and rebase your local branch and then push again

**Comment by mohamedar97** [comment]
@knrt10 the thing is i'm new to git so can please tell me how to do it? or at least direct me to a resource

**Comment by knrt10** [comment]
No problem do this. Make sure this PR is from a fork. If not then

- Create a fork
- clone your fork
- open terminal inside repo
- add another remote name call it  upstream with url as RC original repo url.
**git remote add upstream urlhere**
- git fetch upstream 
- git checkout your branch name
- git rebase upstream/develop

For more info check git documentation on fetch and rebase.

After that run meteor npm install

**Comment by mohamedar97** [comment]
@knrt10  okay i did it, thanks for your patience i'm kinda new to git and still learning

File: app/ui-message/client/messageBox.js [hunk 1]
```diff
@@ -146,6 +146,10 @@ Template.messageBox.onCreated(function() {
 	this.isMicrophoneDenied = new ReactiveVar(true);
 	this.sendIconDisabled = new ReactiveVar(false);
 	messageBox.emit('created', this);
+	window.addEventListener('keydown',function(e){
```
Comment (sampaiodiego) [inline app/ui-message/client/messageBox.js]: I'm not sure this is the best place to do this code. @ggazzo / @tassoevan can you please advise the best place to do this?

@mohamedar97 also this code is not following our code style. please run `meteor npm run lint` before pushing your changes.

**Pull request merged**